scripts/remove_boilerplate.py

```diff
@@ -214,6 +252,12 @@ def main():
     logging.getLogger('trafilatura').setLevel(logging.ERROR)
     install_mp_handler()
 
+    if args.token_filtering and args.token_whitelist:
+        global whitelist
```

Comment:
You don't need `global` here either as you're using the global object, not giving the global variable a new value.

Reply:
Ok.